<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DEDALO serial monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>

      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Serial port and command to send</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Serial port</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_port"/>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="btn_close">
               <property name="text">
                <string>Close port</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Enter</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_cmd"/>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="btn_send">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>

        <item>
         <widget class="QGroupBox" name="console">
          <property name="title">
           <string>Console</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="1">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="2">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="3">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="4">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="5">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="6">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="7">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="8">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="9">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="10">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="11">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="12">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="13">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="14">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="15">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="16">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="17">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="18">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="19">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="20">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="21">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="22">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="23">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="24">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="25">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="26">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="27">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
             <item row="28">
              <widget class="QLabel">
               <property name="text">
                <string> </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
